src/fetch.ts

```diff
@@ -1,7 +1,7 @@
-import { isBrowser, isWorker } from "./utils";
+import { isLongLivedApp, isWorker } from "./utils";
 import fetchShim from "./fetch-shim";
 
-const exported = isBrowser()
+const exported = isLongLivedApp()
```

Comment:
No sure about this one.
Here we don't care about being long lived, just about having native `fetch`. ReactNative doesn't have a `window` right?

Reply:
As discussed in Slack, React Native has a native `window.fetch` 🙂 
https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Core/setUpGlobals.js
https://github.com/facebook/react-native/blob/0b8fd748ee7058a87a5b51959b2344f4df7ed39e/packages/react-native/types/modules/globals.d.ts#L55